miden-lib/asm/miden/kernels/tx/prologue.masm

```diff
@@ -544,6 +544,16 @@ proc.process_input_note
     exec.memory::set_consumed_note_metadata
     # => [note_ptr]
 
+    # ingest note args
+    # ---------------------------------------------------------------------------------------------
```

Comment:
If we are adding this section separator, we should probably add a similar one for `num_inputs` and `num_assets` below.